test/rules/sort-objects.test.ts

```
@@ -1611,61 +1611,19 @@ describe('sort-objects', () => {
         output: dedent`
           let obj = {
             method() {},
-            multilineProperty: {
-              // Some multiline stuff
-            },
+            memberProperty: something,
           }
         `,
         code: dedent`
```

ESLint

[error] Unsafe use of a(n) `error` type typed template tag.

(typescript/no-unsafe-call)